spec/models/google_analytics/data_tracking_spec.rb

```diff
@@ -40,53 +54,47 @@ module GoogleAnalytics
       end
     end
 
-    describe 'type methods' do
-      before do
-        allow(described_class).to receive(:adapter).and_return('Adapter')
-        allow(Rails).to receive(:env).and_return('production'.inquiry)
-      end
+    describe '#tag_manager?' do
```

Comment:
🎖️ much better than inappropriate describe block description

Reply:
I actually deleted this `describe` and changed the `context` on what was line 49 to a `describe`. Most of this section is just whitespace changes after removing one level of nesting.